src/__tests__/01b-create-users.ts

```diff
@@ -132,9 +134,33 @@ describe("username tests", () => {
     await expect(userWallet2.setUsername({ username })).rejects.toThrow()
   })
 
-  // FIXME: failing for some reason
-  // it('sets account status correctly', async () => {
-  //   await setAccountStatus({uid: userWallet2._id, status: 'locked'})
-  //   await expect(userWallet2.status).toBe('locked')
-  // })
+  it("sets account status correctly", async () => {
```

Comment:
name of the test seems outdated

Reply:
Nope. The test checks if the account status is set correctly by the `setAccountStatus` function.